src/main/java/dev/pgm/community/requests/commands/SponsorCommands.java

```diff
@@ -49,29 +42,17 @@
 import tc.oc.pgm.util.named.NameStyle;
 import tc.oc.pgm.util.text.TextFormatter;
 
-@CommandAlias("sponsor")
-@Description("View the sponsor request menu")
 public class SponsorCommands extends CommunityCommand {
 
-  @Dependency private RequestFeature requests;
-  @Dependency private UsersFeature users;
+  private static final String CMD_NAME = "sponsor";
```

Comment:
this can just be a `@CommandMethod("sponsor")` on the class level, which avoids you needing the constant and concatenation in each method